pest-ion/tests/cli.rs

```diff
@@ -16,8 +17,49 @@ enum FileMode {
     Named,
 }
 
+/// Simple wrapper for a CLI test case.
+struct TestCase<S: AsRef<str>> {
+    /// The text of the Pest grammar to test
+    pest_text: S,
+    /// The expected Ion
+    expected_ion: OwnedElement,
+}
+
+impl<S: AsRef<str>> TestCase<S> {}
```

Comment:
What's this empty `impl` block doing?

Reply:
Nothing--this was a remnant from when I was going to make a `new`.